xml/security_kerberos.xml

```diff
@@ -1837,25 +1847,46 @@ objectClass: nsSaslMapping</screen>
     </para>
    </step>
   </procedure>
-
-  <para>
-   <remark>taroth 2019-11-13: FIXME: what are we doing with the following steps?
-   are the commands consistent with the examples we used so far or do we
-   need to replace some strings?</remark>
 <!-- info by wbrown, via mail (2019-07-02)-->
-   </para>
-
-<screen>after you follow the Yast KDB setup steps, you can then do:
-
-# kadmin.local
-kadmin.local: listprincs
-kadmin.local: ank admin@EXAMPLE.COM
-
-
-# ldapsearch -D 'cn=Directory Manager' -w password -b 'cn=EXAMPLE.COM,cn=kdc,dc=example,dc=com' -H ldaps://localhost
-
-
-# admin@EXAMPLE.COM, EXAMPLE.COM, kdc, example.com
+  <para>
+   To check if the setup of the Kerberos back-end inside of LDAP was successful,
+   proceed as follows:
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Directly accesses the KDC database:
+    </para>
+    <screen>&prompt.sudo;kadmin.local</screen>
+   </step>
+   <step>
+    <para>
+     List the principals:
+    </para>
+    <screen>kadmin.local&gt; listprincs</screen>
+   </step>
+   <step>
+    <para>
+     Create a principal:
+     <remark>@firstyear: a principal with an admin role has
+      already been created in section 6.5.5.3 - do we need another one here?</remark>
+    </para>
+    <screen>kadmin.local&gt; ank admin@EXAMPLE.COM</screen>
+   </step>
+   <step>
+    <para>
+     <remark>@firstyear: what are we doing in this step and why? and why do we use
+      'ldapsearch', when we advised _against_ its use in section 5.6
```

Comment:
because 389-ds upstream won't (and will never) ship krb management tools, so we have to use the ldap search command here.